core/src/main/java/com/gcinterceptor/core/GarbageCollectorControlInterceptor.java

```diff
@@ -58,6 +67,30 @@ private ShedResponse shed() {
 		return new ShedResponse(true);
 	}
 
+	public void saveInCSV(String line) {
+		FileWriter fw = null;
+		BufferedWriter bw = null;
+
+		try {
+			fw = new FileWriter(csv_file_name, true);
```

Comment:
Did you check what happen when you open and close the filewriter and bufferedWriter at every write. This sounds super slow to me, thus I believe that we are going to have an performance impact on our experiments.

Could you please check if it is worth to optmitze?

Reply:
Example of library and some other compared libraries.

Reply:
Gonna check it out.